config/jest/setup.js

```diff
@@ -6,13 +6,14 @@ jest.unmock('object-assign');
 
 require('../polyfills');
 
-const enzyme = require.requireActual('enzyme');
-const Adapter = require.requireActual('enzyme-adapter-react-15');
-
-// Configure enzyme to work with React 15 for now
-enzyme.configure({ adapter: new Adapter() });
-
 // Starting with React 16, we'll have to polyfill this in test environments.
+// Defining this earlier than loading Enzyme to avoid "React depends on requestAnimationFrame" warning.
 global.requestAnimationFrame = function(callback) {
```

Comment:
Want to move this chunk to `polyfills.js` actually? That way it's a part of that `require` statement above.

Reply:
Good call, done!